src/docs/product/sentry-basics/search/searchable-properties/index.mdx

```diff
@@ -0,0 +1,1158 @@
+
+- **Property of:** events, issues, replays
+- **Type:** uuid
+
+### `trace.parent_span`
+
+Span ID of the parent to the current transaction. This is null if the transaction is root.
+
+- **Property of:** events
+- **Type:** uuid
+
+### `trace.span`
+
+Span ID of the root span of the root transaction in the event.
+
+- **Property of:** events
+- **Type:** uuid
+
+### `transaction`
+
+For [transactions](/product/performance/transaction-summary/#what-is-a-transaction), the name of the transaction. For errors, the name of the associated transaction.
+
+- **Property of:** events, issues
+- **Type:** string
+
+### `transaction.duration`
+
+Duration, in milliseconds, of the transaction.
+
+- **Property of:** events
+- **Type:** duration
+
+### `transaction.op`
+
+Short code identifying the [type of operation](https://develop.sentry.dev/sdk/performance/span-operations/) the span is measuring.
+
+- **Property of:** events
+- **Type:** string
+
+### `transaction.status`
+
+Describes the status of the span/transaction. Check out our [Transaction Payloads documentation](https://develop.sentry.dev/sdk/event-payloads/transaction/) for all possible statuses.
+
+- **Property of:** events
+- **Type:** string
+
+### `unreal.crash_type`
+
+The [Unreal Crash Context Type](https://docs.unrealengine.com/4.27/en-US/API/Runtime/Core/GenericPlatform/ECrashContextType/)
+
+- **Property of:** events, issues
+- **Type:** string
+
+### `url`
+
+A specific URL that the user visited during the replay.
+
+- **Property of:** replays
+- **Type:** string
+
+### `user_misery(number)`
+
+Returns transactions with the defined user misery value. [User Misery](/product/performance/metrics/#user-misery) is a user-weighted performance metric that counts the number of unique users who were frustrated; "frustration" is measured as a response time four times the satisfactory response time threshold (in milliseconds). It highlights transactions that have the highest impact on users.
+
+- **Property of:** events
+- **Type:** number
+
+### `user.display`
+
+In order, the first available user field available: email, then username, ID, and then IP address.
+
+- **Property of:** events
+- **Type:** string
+
+### `user.email`
+
+An alternative, or addition, to the username. Sentry is aware of email addresses and can therefore display things such as Gravatars and unlock messaging capabilities.
+
+- **Property of:** events, issues, replays
+- **Type:** string
+
+### `user.id`
+
+Application-specific internal identifier for the user.
+
+- **Property of:** events, issues, replays
+- **Type:** string
+
+### `user.ip`
+
+User's IP address. Sentry uses the IP address as a unique identifier for unauthenticated users.
+
+- **Property of:** events, issues, replays
+- **Type:** string
+
+### `user.username`
+
+Username, which is typically a better label than the `user.id`.
+
+- **Property of:** events, issues, replays
+- **Type:** string
+
+</DefinitionList>
+
+## Custom Tags
```

Comment:
I think it could help to move this section & device classification up before the list of all properties so they're better surfaced to readers. Putting them below the long list makes it easy to miss.

Reply:
Due to the removal of the combined list, this will no longer be necessary.